ui/selectors/multichain-accounts/account-tree.ts

```diff
@@ -41,10 +41,11 @@ import { getSanitizedChainId, extractWalletIdFromGroupId } from './utils';
  * @param state - Redux state.
```

Comment:
Moved to a separate PR https://github.com/MetaMask/metamask-extension/pull/35381